### PR TITLE
Create index.html initial structure

### PR DESCRIPTION
**Pull request by hola-soy-milk**
Getting started with our website, this will create the `index.html` file, and place it in the main folder of the project.

Any feedback is super welcome, thank you! ☺️

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (hola-soy-milk) [inline index.html:2]: Nice! Love that we're including a language specification 👍

Comment (jessicarose) [reply index.html:2]: Isn't this so nice?

**Review by jessicarose — APPROVED** [review]
Great work, are we going to add the css file next?

**Comment by hola-soy-milk** [comment]
That's the plan, thanks for reviewing! 🙌

**Pull request merged**